selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -32,7 +32,6 @@
           <class name="org.eclipse.che.selenium.languageserver.TypeScriptEditingTest"/>
           <class name="org.eclipse.che.selenium.languageserver.ClangFileEditingTest"/>
           <class name="org.eclipse.che.selenium.languageserver.ApacheCamelFileEditingTest"/>
-          <class name="org.eclipse.che.selenium.languageserver.YamlFileEditingTest"/>
```

Comment:
Please, don't forget to remove test class if it is being removed from test suite.

Reply:
Done